.chloggen/ottl-lists.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: pkg/ottl
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add lists to the OTTL grammar and change the function signature of `Concat`.
+
+# One or more tracking issues related to the change
+issues: [13391]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Existing variadic function calls should be updated to put the parameters in a list.
```

Comment:
Let's list out all the functions affected.

Reply:
Good callout, I've added this to the subtext.